Microsoft.Quantum.Canon/Utils/Multiplexor.qs

```diff
@@ -0,0 +1,285 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think that even though "multiplexor" is a correct spelling, "multiplexer" is enough more common that it might make sense to use "-xer" in preference to "-xor."

Reply:
Renamed all instances of "multiplexor" to "multiplexer".